### PR TITLE
Fix input error styling

### LINKED ISSUE #1954: `LabelWrapper` styling needs to updated
Input error text doesn't match design. See: https://github.com/decentdao/fractal-interface/issues/1600

### LINKED ISSUE #1970: Template create page - Desktop - Show/Hide in description preview is truncated immediately
As soon as I start typing a description, the preview link to show/hide immediately becomes truncated. Please update to use the basic text link pattern and not the pill button.

Image

### PR DESCRIPTION
**Pull request by DarksightKellar**
The full error state styling was missing in nearly every input component across the app. This PR fixes that. Closes #1954.

To test, try entering expected invalid input into as many input boxes as you can find. Confirm that the error state UI matches design:

Screenshot 2024-06-05 at 12 26 04

...

Brain fart note: I accidentally committed the fix to closes #1970, to this branch. It's a small change (last 2 commits), so please forgive the added overhead and review in this PR as well 🙏🏾
Screenshot 2024-06-05 at 14 23 10

@tomstuart123 sorry to bring you back to this, but see above added comment

### CODE REVIEW

File: src/components/DaoCreator/formComponents/VotesTokenImport.tsx [hunk 1]
```diff
@@ -48,6 +48,7 @@ export function VotesTokenImport(props: ICreationStepProps) {
         helper={t('helperTokenSupply')}
         isRequired
       >
+        {/* @todo: this allows alphanumeric input. intentional? */}
```
Comment (DarksightKellar) [inline src/components/DaoCreator/formComponents/VotesTokenImport.tsx:51]: `Input` below should probably be `BigIntInput`, but a quick change messed with `onChange` being unable to update value. If `Input` was not intentional, another issue should be created to fix

File: src/i18n/locales/en/daoCreate.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "buttonCreate": "Create a Decent Safe",
   "addNFTButton": "Import another token",
   "labelCreateSubDAOProposal": "Create a child Safe",
+  "errorMinSigners": "Number of signers must be greater than 0",
```
Comment (DarksightKellar) [inline src/i18n/locales/en/daoCreate.json:5]: Reference to this string was in codebase, but somehow never here

**Comment by tomstuart123** [comment]
Hi @DarksightKellar , nice job picking this up. I went through the inputs I could think of and compared side by side with the current production app. the alert icon is slic

Question though - in two locations of the app, the input field is highlighted red BEFORE I click in the input and write anything incorrect. i.e. as soon as I navigate to the page in this PR, I am alerted the input is missing. This was not the case before in the production app

Is this intended functionality?

See examples for you to replicate below during proposal creation and then during token-voting safe creation

Screenshot 2024-06-05 at 12 54 42

Screenshot 2024-06-05 at 12 53 18

**Comment by DarksightKellar** [comment]
> mples for you to replicate below during proposal creation and then during token-voting safe cre

@tomstuart123 
On the token allocation input, [this discussion](https://github.com/decentdao/decent-interface/issues/1738) concluded to leave as is. However, since the error state is a bit more loud, it's definitely much more concerning, so I've made a quick update to separate the "no value" look from the "actual error" look:

Screenshot 2024-06-05 at 13 45 22

Screenshot 2024-06-05 at 13 45 37

On your second point, that was a bug and should be fixed now. Thanks for catching that

**Comment by tomstuart123** [comment]
Thanks @DarksightKellar . Reviewed both the bug and the copy change and like them a lot.

**Review by tomstuart123 — APPROVED** [review]
approved

File: src/components/DaoCreator/formComponents/VotesTokenImport.tsx (hunk 1 above)
Comment (Da-Colon) [reply src/components/DaoCreator/formComponents/VotesTokenImport.tsx:51]: Yeah I agree with you, I don't think it was intentional

File: src/components/ui/forms/CustomNonceInput.tsx [hunk 3]
```diff
@@ -69,10 +69,11 @@ export function CustomNonceInput({
           )
         }
         testId={`custom-nonce`}
+        isInvalid={!!errorMessage}
       />
       {errorMessage && (
         <Text
-          color="alert-red.normal"
+          color="red-0"
```
Comment (Da-Colon) [inline src/components/ui/forms/CustomNonceInput.tsx:76]: ah leftovers

**Review by Da-Colon — APPROVED** [review]
fantastic, code looks good

**Comment by tomstuart123** [comment]
@DarksightKellar - reviewed the fix for 1970 in both proposal creation and template builder

Both work well. Approved

**Review by tomstuart123 — APPROVED** [review]

**Pull request merged**